index.html

```diff
@@ -0,0 +1,160 @@
+
+    <main class="content">
+      <!--added id element in order to link the search engine bar on the nav bar -->
+
+      <article
+        class="search-engine-optimization"
+        id="search-engine-optimization"
+      >
+        <figure class="services-img">
+          <img
+            src="./assets/images/search-engine-optimization.jpg"
+            class="float-left"
+            alt="Birds eye view of a desk with a laptop,cup of tea and pens"
+          />
+          <figcaption>
+            Birds eye view of a desk with a laptop,cup of tea and pens
+          </figcaption>
+        </figure>
+
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </article>
+
+      <article
+        id="online-reputation-management"
+        class="online-reputation-management"
+      >
+        <figure>
+          <img
+            src="./assets/images/online-reputation-management.jpg"
+            class="float-right"
+            alt="image of laptop showing a gr"
+          />
+
+          <figcaption>Laptop representing data</figcaption>
+        </figure>
+
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </article>
+
+      <article id="social-media-marketing" class="social-media-marketing">
+        <figure>
+          <img
+            src="./assets/images/social-media-marketing.jpg"
+            class="float-left"
+            alt="birds eye view of a table with a laptop and text of different methods of social media marketing"
+          />
+          <figcaption>Different methods of social media marketing</figcaption>
+        </figure>
+
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </article>
+    </main>
+
+    <!--Semantic HTML: The additional content on the right hand side is nested in a aside tag-->
+
+    <!-- Semantic HTML: Due to accessability alt attribute is added to all the images in the aside section-->
+
+    <aside class="benefits">
+      <div class="benefit-lead">
```

Comment:
A section tag could have been used here instead of a `div`. See [here](https://medium.com/design-code-repository/html-elements-section-vs-div-vs-article-a8c34e6548cf) for more information about `section` vs `div`. Apply this to all other sections.